media/doc/WINESYNC.txt

```diff
@@ -280,20 +280,18 @@ iphlpapi -
   modules/rostests/winetests/iphlpapi # Synced to WineStaging-1.9.11
 
 kernel32 -
-  dll/win32/kernel32/wine/actctx.c               # Partly synced with Wine 1.7.55
-  dll/win32/kernel32/wine/comm.c                 # Synced in r52754
+  dll/win32/kernel32/wine/actctx.c               # Synced to wine-3.3
+  dll/win32/kernel32/wine/comm.c                 # Synced to wine-3.3
   dll/win32/kernel32/wine/lzexpand.c             # Synced to WineStaging-3.3
   dll/win32/kernel32/wine/profile.c              # Synced to WineStaging-3.3
   dll/win32/kernel32/wine/res.c                  # Partially synced to WineStaging-3.3
   dll/win32/kernel32/winnls/string/casemap.c     # Synced to WineStaging-3.3
-  dll/win32/kernel32/winnls/string/chartype.c    # Synced in r52754
-  dll/win32/kernel32/winnls/string/collation.c   # Synced to WineStaging-1.7.55
+  dll/win32/kernel32/winnls/string/collation.c   # Synced to WineStaging-3.3
   dll/win32/kernel32/winnls/string/digitmap.c    # Synced to WineStaging-3.3
   dll/win32/kernel32/winnls/string/fold.c        # Synced to WineStaging-3.3
   dll/win32/kernel32/winnls/string/format_msg.c  # Synced to WineStaging-3.3
-  dll/win32/kernel32/winnls/string/lang.c        # Synced in r52754
+  dll/win32/kernel32/winnls/string/locale.c      # Synced in wine-3.3
   dll/win32/kernel32/winnls/string/lcformat.c    # Synced to WineStaging-3.21
```

Comment:
Want to upgrade this file to WineStaging-3.3?

Reply:
No